imod_coupler/utils.py

```diff
@@ -13,7 +13,7 @@
 
 
 def create_mapping(
-    src_idx: Any, tgt_idx: Any, nsrc: int, ntgt: int, operator: str
+    src_idx: Any, tgt_idx: Any, nsrc: int, ntgt: int, operator: str, weight: Optional[NDArray] = None
```

Comment:
for the operator string  you could use an enum:
from enum import Enum
 class Operation(Enum):
      Sum = auto()
      Average = auto()
      Weigths = auto()

Reply:
this Optional specification was added by Julian. Can i keep it like this?

Reply:
Luit Jan meant the operator, not the weight.
I agree, but would suggest [`StrEnum`](https://docs.python.org/3/library/enum.html#enum.StrEnum) instead

Reply:
Correction: Luit-Jan's suggestion is better.

Reply:
Done